app/assets/javascripts/components/components/status.jsx

```diff
@@ -53,6 +53,7 @@ class Status extends React.PureComponent {
         <div className='status__wrapper'>
           <div className='status__prepend'>
             <div className='status__prepend-icon-wrapper'><i className='fa fa-fw fa-retweet status__prepend-icon' /></div>
+            <Avatar src={status.getIn(['account', 'avatar'])} staticSrc={status.getIn(['account', 'avatar_static'])} size={20} style={{ display: 'inline-block', verticalAlign: 'middle' }} />
```

Comment:
Styles should be written in [`components.scss`](https://github.com/tootsuite/mastodon/blob/0dd71af5fab7d2bef59a6e9d467b2742476aee00/app/assets/stylesheets/components.scss). I think it would be better to have a screenshot.